Exchange-Guide.md

```diff
@@ -0,0 +1,179 @@
+
+Once the setup is completed, you will be able to query pending orders, place orders, and experiment with `xud`'s rich set of commands. Once a match is found in the network, your orders will be executed via a cross-chain atomic swap. The XUD simulation network currently supports atomic swaps between BTC and LTC. This guide will show how to to install GO, bitcoin and litecoin daemons, setup Lightning daemons (LND) for bitcoin (BTC) and litecoin (LTC), setup `xud` and basic instructions of how to integrate its API and finally connect to the simulation network. This guide is written for Linux & MacOS operating systems and geared towards IT administrators of exchanges.
+
+## Describing the Setup
+To connect to the XUD simulation network you'll need to run several components which together provide support for exchanging order information and atomic swaps between BTC and LTC on the lightning network. You will setup the following components:
+- `BTCD` - full node, connected to the BTC chain
+- `LTCD` - full node, connected to the LTC chain
+- `LND-BTC` - lightning network daemon for the BTC network
+- `LND-LTC` - lightning network daemon for the LTC network
+- `XUD` - Exchange Union's decentralized exchange layer
+
+## Installation 
+### Prerequisites
+Make sure you have the below programs installed:
+- [git](https://gist.github.com/derhuerst/1b15ff4652a867391f03#file-linux-)
+- [make](https://www.cyberciti.biz/faq/debian-linux-install-gnu-gcc-compiler/)
+- [node.js + npm](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions)
+- killall (`sudo apt-get install psmisc`)
+
+### Installing GO
+if GO is not yet installed on your system please use the following command to install it:
+```
+sudo apt-get install golang-1.10-go
+```
+Note that golang-1.10-go puts binaries in /usr/lib/go-1.10/bin. If you want them on your PATH, you need to make that change yourself. Alternatively, you can create a softlink as follows:
+```
+sudo ln -s /usr/lib/go-1.10/bin/go /usr/local/bin/go 
+```
+
+### Repository Checkout
+From your home directory run
+```
+git clone https://github.com/ExchangeUnion/xud-simnet.git
+```
+This should create the xud-simnet directory under your home directory.
+
+### Setup ~/.bashrc
+To enable access to the scripts please update and source your `.bashrc`:
+```
+source ~/xud-simnet/setup.bash
+source ~/.bashrc
+```
+Please note that the `setup.bash` script set your `$GOPATH` to `~/xud-simnet/go`. All changes from `setup.bash`are temporary and only active for the current terminal session. You need to repeat the sourcing above or add things to `PATH` to make these permanent.
+
+### Verify Setup
+Before you move on lets verify that we are ready:
+```
+xud-simnet-check
```

Comment:
@kilrau note that this can be embedded in to xud-simnet-install and removed from here.

Let me know if you like it.

Reply:
Of course, please do that, let me know and I'll update the guide

Reply:
Let me know once this is up so I can adjust the guide @offerm